test/test-internal-properties.js

```diff
@@ -42,7 +49,10 @@ it('can use "ownKeys" api.', () => {
   document.body.append(el);
   const ownKeys = Reflect.ownKeys(el.internal);
   assert(
-    ownKeys.length === 2 && ownKeys[0] === 'internalProperty' && ownKeys[1] === 'internalComputedProperty',
+    ownKeys.length === 3 &&
+    ownKeys[0] === 'internalProperty' &&
+    ownKeys[1] === 'internalComputedProperty' &&
+    ownKeys[2] === 'children',
```

Comment:
This is sorta unrelated — because we added a new property on the test element, we need to also update this test to consider it.